apps/charterafrica/src/lib/data/index.js

```diff
@@ -75,6 +75,33 @@ async function processPageNews({ blocks }) {
 
 async function processPageResearch({ blocks }) {
   // TODO(kilemensi): Pull data from CMS
+
+  blocks.push({
+    slug: "featured-post",
+    title: "Featured Post",
+    article: {
+      id: 1,
+      title: "Featured Post title goes here and spans over second line. ",
+      excerpt:
+        "Lorem ipsum dolor sit amet consectetur adipiscing elit tempus nibh cursus, urna porta sagittis non eget taciti nunc sed felis dui, praesent ullamcorper facilisi euismod ut in platea laoreet integer. Lorem ipsum dolor sit amet consectetur  ",
+      author: "Sakwa G",
+      date: "2023-02-11",
+      image: {
+        id: "63d2622aafe25f6469605eae",
+        alt: `Featured Post`,
+        prefix: "media",
+        filename: "Rectangle 113.jpg",
+        mimeType: "image/jpg",
+        filesize: 257010,
+        width: 1236,
+        height: 696,
+        createdAt: "2023-01-26T11:21:14.868Z",
+        updatedAt: "2023-01-26T11:21:14.868Z",
+        url: "http://localhost:3000/images/featured_post.jpeg",
```

Comment:
```suggestion
        url: "/images/featured_post.jpeg",
```

Use relative paths since it's possible for someone to run the app on different port e.g. I run it under `3003`.